meta/run.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
+
+config_set: 
+   keyvaluestore:
+      bootcmd: /node keyvaluestore.js
+      env:
+         PORT: 9000
+   db:
+      bootcmd: /node db.js
+      env:
+         PORT: 9001
+         MICRO_KEYVALUESTORE_ENDPOINT: {IP}:9000
```

Comment:
Nothing, it's just placeholder that user should manually replace (so the IP address of the keyvaluestore)